styleguide.config.js

```diff
@@ -197,8 +186,6 @@ module.exports = {
     url: 'https://github.com/zopaUK/react-components',
     text: 'Fork me!',
   },
-
-  ignore: ['**/*.test.{js,jsx,ts,tsx}', '**/*.d.ts', '**/src/styleguide-components/**', '**/styleguide/**'],
```

Comment:
this is the default, except for the `src/styleguide-components` but the components are always taken from `src/components` so it's not needed